services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -1926,13 +1927,13 @@ func TestReleaseTrainErrors(t *testing.T) {
 			expectedError: "",
 			expectedCommitMsg: `Release Train to environment/environment group 'acceptance':
 
-Release Train to 'acceptance-ca' environment:
+Release Train to 'acceptance-ca' environment for 0 apps:
```

Comment:
I think this change would get you into conflicts with @hego-fdc 
I would suggest to undo the "for n apps" addition to make merging simpler.